scheduler/src/cook/mesos/util.clj

```diff
@@ -471,23 +471,37 @@
 
 (defn get-user-running-job-ents
   "Returns all running job entities for a specific user and pool."
-  [db user pool-name]
-  (let [requesting-default-pool? (or (nil? pool-name) (default-pool? pool-name))
-        pool-name' (or pool-name (config/default-pool) (UUID/randomUUID))]
-    (timers/time!
-      get-user-running-jobs-duration
-      (->> (q
-             '[:find [?j ...]
-               :in $ ?user ?pool-name ?requesting-default-pool
-               :where
-               ;; Note: We're assuming that many users will have significantly more
-               ;; completed jobs than there are jobs currently running in the system.
-               ;; If not, we might want to swap these two constraints.
-               [?j :job/state :job.state/running]
-               [?j :job/user ?user]
-               [(cook.mesos.pool/check-pool $ ?j :job/pool ?pool-name ?requesting-default-pool)]]
-             db user pool-name' requesting-default-pool?)
-           (map (partial d/entity db))))))
+  ([db user]
```

Comment:
Does this make sense to split into a separate function? I think intuitively a 2 argument version of this function would return the jobs in the default pool, not all jobs. In any case, we should update the docstring.

Reply:
Done